crates/holochain_zome_types/src/entry.rs

```diff
@@ -83,7 +83,13 @@ impl Default for GetOptions {
     }
 }
 
-#[derive(PartialEq, Debug, Clone, Copy, Serialize, Deserialize)]
+impl From<GetStrategy> for GetOptions {
```

Comment:
Seems reasonable. This has been the only option for a good long time and if we add anything to it and want it to be forward compatible then we have to make the new fields optional anyway.